cmd/policy_describe.go

```diff
@@ -0,0 +1,29 @@
+package cmd
+
+import (
+	"context"
+
+	"github.com/cloudquery/cloudquery/pkg/ui/console"
+	"github.com/spf13/cobra"
+)
+
+const policyDescribeHelpMsg = `Describe CloudQuery policy`
+
+var (
+	describePolicyCmd = &cobra.Command{
+		Use:   "describe",
+		Short: policyDescribeHelpMsg,
+		Long:  policyDescribeHelpMsg,
+		Args:  cobra.ExactArgs(1),
+		Run: handleCommand(func(ctx context.Context, c *console.Client, cmd *cobra.Command, args []string) error {
+			_ = c.DescribePolicies(ctx, args, args[0], skipVersioning)
```

Comment:
I am not sure I fully understood what we want to do with this, since it is only used on the module caller, I think the error handling inside the client/policy is lacking and we should address that part first before doing a sweeping change on the ConsoleError.

Reply:
We can reference this request in a separate issue and fix this in a later PR.